tests/mktree.common

```diff
@@ -24,6 +24,12 @@ make_install()
     mkdir -p $DESTDIR/build
     ln -sf ../data/SOURCES $DESTDIR/build/
 
+    # setup an empty db that all tests are pointed to by default
+    dbpath="/var/lib/rpm-testsuite"
```

Comment:
Should probably use `local` here (to avoid leaking the variable outside this bash function).

Reply:
I'm also thinking that for consistency, we could name this `/var/lib/mktree` or something (as there's already `/usr/share/mktree` for some helper scripts). But that's really the absolutely last thing to worry about here :smile:

Reply:
I dislike the rpm-testsuite name too, but I do want it to have rpm in it :sweat_smile:

Reply:
/var/lib/rpm-testdb ?

Reply:
OTOH this is now centrally set, we only need to change that one line to move it elsewhere.

Reply:
Nah, I don't really care that much :smile: Let's keep it as is (whichever name works, really).